build/build.xml

```diff
@@ -254,13 +258,17 @@
   </macrodef>
 
   <!-- copy hardware folder -->
-  <target name="assemble-hardware" unless="light_bundle">
+  <target name="assemble-hardware" depends="assemble-hardware-full, assemble-hardware-light" />
+  <target name="assemble-hardware-full" unless="light_bundle">
     <copy todir="${target.path}/hardware">
       <fileset dir="../hardware">
         <exclude name="arduino/sam/**"/>
       </fileset>
     </copy>
   </target>
+  <target name="assemble-hardware-light" if="light_bundle">
+    <copy file="../hardware/package_index_bundled.json" todir="${target.path}/hardware" />
```

Comment:
In this case the file `../hardware/package_index_bundled.json` is wrong, it should be a json with zero platforms defined. I don't know maybe the best thing to do is to add a check to the IDE...